app/components/agenda/compare-agenda-list.js

```diff
@@ -7,8 +7,9 @@ import { tracked } from '@glimmer/tracking';
 export default class CompareAgendaList extends Component {
   /**
    * INFO arguments from parent.
-   * @reversedAgendas all agendas from the meeting reverse sorted on serial number
-   * @isShowingChanges flag to show all agendaitems or just the ones with changes
+   * @argument meeting the current meeting
+   * @argument reversedAgendas all agendas from the meeting reverse sorted on serial number
```

Comment:
The need for the reverse sorting is specific to this component (0functionality), right? Would it be better to move the sorting responsibility inside the component?

Reply:
It may not be clear from this PR, but reversedAgendas is used in components.
- Agenda-actions
- side-nav
- agendaitem-controls
- compare-agenda-list

Reply:
It may not be clear from this PR, but reversedAgendas is used in 4 components.
- Agenda-actions
- side-nav
- agendaitem-controls
- compare-agenda-list